ods_ci/tests/Resources/OCP.resource

```diff
@@ -143,7 +143,7 @@ Wait For Pods To Succeed
 
 Wait For Pods To Be Terminated
     [Arguments]    ${label_selector}    ${namespace}    ${timeout}=180s
-    Wait Until Keyword Succeeds    ${timeout}    3s
+    Wait Until Keyword Succeeds    ${timeout}    15s
```

Comment:
better to parametrize it. The retry depends on the timeout as well, this may impact other test cases